### DISCUSSION TITLE
V4.6.1

### DISCUSSION THREAD

**Discussion by c121914yu** [labring/FastGPT#500] [Announcements]
1. 新增 - GPT4-v 模型支持
2. 新增 - whisper 语音输入
3. 优化 - TTS 流传输
4. 优化 - TTS 缓存

<hr /><em>This discussion was created from the release <a href='https://github.com/labring/FastGPT/releases/tag/v4.6.1'>V4.6.1</a>.</em>

**Comment by qsimu** [comment]
![image](https://github.com/labring/FastGPT/assets/42400779/4d9b7914-991a-4288-bb9f-d53628752385)
![image](https://github.com/labring/FastGPT/assets/42400779/e28a3d62-e082-4ecb-b7bd-8ce483b05bc0)
![image](https://github.com/labring/FastGPT/assets/42400779/37081265-7501-4c26-8b53-69ac92f2ec18)
是我不会用吗, 4v模型好像不能用呀.

**Reply by c121914yu** [reply]
是本地？本地的话无法测试，因为无法读取localhost的地址

**Reply by qsimu** [reply]
这个是在容器里.
